templates/partial/search_result.html

```diff
@@ -18,10 +18,16 @@ <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
           {% endif %}
       </h3>
       {% if result.description %}
-      <div class="govuk-body-m">{{result.metadata.search_summary|markdown}}</div>
+      <div class="govuk-body-m">
+        {% if result.description|length > 200 %}
+          {{ result.description|slice:":200"|add:"..."|markdown }}
```

Comment:
A character-based slice might cause problems here depending on where it ends up slicing.  Looks like `truncatewords` might work better for this:

https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#truncatewords